library/core/src/panicking.rs

```diff
@@ -79,14 +80,34 @@ pub fn panic_fmt(fmt: fmt::Arguments<'_>) -> ! {
         super::intrinsics::abort()
     }
 
+    start_panic(None, &fmt, None)
```

Comment:
Now that `panic_fmt` just directly calls `start_panic`, there's not much need to check for `panic_immediate_abort` here or to not inline it.